websites/N/Neovel/dist/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Neovel",
+	"description": {
+		"en": "Designed by and for book lovers ! Discover countless new stories at your fingertips and join a strong positive community !",
+		"fr": "Conçu par et pour les amoureux des livres ! Découvrez d'innombrables nouvelles histoires à portée de main et rejoignez une forte communauté positive !",
+		"es": "¡Hecho, diseñado y pensado para los amantes de las novelas! ¡Con la punta de tus dedos puedes acceder, leer & descubrir montones de historias nuevas y una comunidad fuerte!"
```

Comment:
The ampersand (&) is very rarely, if its ever, used in Spanish. Please, change it by "y".

Reply:
Resolved on 9729af17ed25f3d98a94f23e43dbc9ec23c31230